### DISCUSSION TITLE
Add some MCQ

### DISCUSSION THREAD

**Discussion by MonadWizard** [rkshaon/software-engineering-preparation#29] [Q&A] [answered]
for python 
https://monadwizard.github.io/blog-v2/pyQ1.html

for javascript 
https://monadwizard.github.io/blog-v2/jsQ50.html

for blog 
https://monadwizard.github.io/blog-v2/

**Comment by rkshaon** [comment, marked as answer]
Thanks for your recommendation, will be added very soon.
